docs/guides/enqueue.md

```diff
@@ -45,6 +48,42 @@ Class constants:
   const ADMIN_STYLE_URI  = 'applicationAdmin.css';
 ```
 
+In order to override this class, you should create a new class, preferably in the `enqueue` folder in your project. For instance, if you'd like to remove the login and admin styles from your project, in your `enqueue\class-admin-enqueue.php` you'd do:
+
+```php
```

Comment:
hm, just a thought would it be smarter to have a new docs section for example "Remove built in action" and explain there how removing actions work for all classes. You can reuse this example there.

My thought is that we can explain how to do something in one place and not in multiple sections.

what do you think?

Reply:
> hm, just a thought would it be smarter to have a new docs section for example "Remove built in action" and explain there how removing actions work for all classes. You can reuse this example there.
> My thought is that we can explain how to do something in one place and not in multiple sections.
> what do you think?

I'd agree although I'd just add a new `h2` in this file `enqueue.md` called "Remove built in actions" and put your example there. Currently if you're specifically looking how to remove actions you'll never find it unless you happen to stumble upon it.